internal/experiments/experiments.go

```diff
@@ -55,12 +54,13 @@ var (
 	}
 
 	Promoted = map[string]string{
-		ANSITimestamps:     standardPromotionMsg(ANSITimestamps, "v3.48.0"),
-		AvoidRecursiveTrap: standardPromotionMsg(AvoidRecursiveTrap, "v3.66.0"),
-		FlockFileLocks:     standardPromotionMsg(FlockFileLocks, "v3.48.0"),
-		GitMirrors:         standardPromotionMsg(GitMirrors, "v3.47.0"),
-		InbuiltStatusPage:  standardPromotionMsg(InbuiltStatusPage, "v3.48.0"),
-		JobAPI:             standardPromotionMsg(JobAPI, "v3.64.0"),
+		ANSITimestamps:         standardPromotionMsg(ANSITimestamps, "v3.48.0"),
+		AvoidRecursiveTrap:     standardPromotionMsg(AvoidRecursiveTrap, "v3.66.0"),
+		FlockFileLocks:         standardPromotionMsg(FlockFileLocks, "v3.48.0"),
+		GitMirrors:             standardPromotionMsg(GitMirrors, "v3.47.0"),
+		InbuiltStatusPage:      standardPromotionMsg(InbuiltStatusPage, "v3.48.0"),
+		IsolatedPluginCheckout: standardPromotionMsg(IsolatedPluginCheckout, "v3.67.0"),
+		JobAPI:                 standardPromotionMsg(JobAPI, "v3.64.0"),
```

Comment:
should we sort these by version number instead of alphabetically?

Reply:
I think that's a good idea, but enforcing it will not be straight forward at the moment. Always happy to review a PR by you that does it.

Reply:
Why by version number? Not really seeing a reason

Reply:
eh, i was thinking to sort by recency? but also it really doesn't matter that much, i think